app/serializers/finance/npq/assurance_report/csv_serializer.rb

```diff
@@ -68,6 +69,7 @@ def to_row(record)
             record.declaration_created_at.iso8601,
             record.statement_name,
             record.statement_id,
+            record.targeted_delivery_funding_eligibility,
```

Comment:
I think you'll also need to add:
```
a. targeted_delivery_funding_eligibility
```
here as well: https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/services/finance/npq/assurance_report/query.rb#L28